helm/cloud-provider-vsphere/values.yaml

```diff
@@ -76,31 +83,3 @@ daemonset:
   nodeSelector: {}
   ## Allows for the default tolerations to be replaced with user-defined ones
   tolerations: []
-
-# Service configuration
-service:
```

Comment:
Not sure what these were used for but fine if we don't need em.

Reply:
well, they are gone in the upstream helm chart. The less yaml, the better :D

Reply:
Amen to that